tests/FakeItEasy.Tests/Creation/CastleDynamicProxy/CastleDynamicProxyGeneratorTests.cs

```diff
@@ -207,7 +212,11 @@ public void Should_specify_that_no_default_constructor_was_found()
             result.ReasonForFailure.Should().StartWith("No usable default constructor was found on the type");
         }
 
+#if !NETCORE
         [Fact]
+#else
+        [Fact(Skip = "System.AppDomainInitializerInfo is not available in .NET Core. Furthermore, even if we could found another non-static internal class in mscorlib that is type-forwarded to Private.CoreLib on .NET Core it may disappear in the future too.")]

```

Comment:
I found one that worked `"FluentAssertions.Common.NullReflector, FluentAssertions.Core"`